src/sentry/integrations/jira_server/webhooks.py

```diff
@@ -1,8 +1,48 @@
 from __future__ import absolute_import
 
+import jwt
+import logging
+import six
 from django.views.decorators.csrf import csrf_exempt
 
 from sentry.api.base import Endpoint
+from sentry.integrations.jira.webhooks import (
+    handle_assignee_change,
+    handle_status_change
+)
+from sentry.models import Integration
+
+
+logger = logging.getLogger('sentry.integrations.jiraserver.webhooks')
+
+
+def get_integration_from_token(token):
```

Comment:
@markstory is this significantly different from https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/atlassian_connect.py#L42

Reply:
Yes and no. Its the same basic approach as I didn't want to totally re-invent the wheel. However, this doesn't generate/validate a query hash as it isn't something we can reasonably do with jira-server.